app/controllers/restaurants_controller.rb

```diff
@@ -40,4 +48,22 @@ def set_restaurant
     def restaurant_params
       params.require(:restaurant).permit(:name, :cuisine, :rating, :ten_bis, :address, :max_delivery_time)
     end
+
+  def create_where_params_obj(query_params)
+    params = {}
+
+    if query_params['rating'].present?
+      params['rating'] = (query_params['rating'].to_i)..Restaurant::MAX_RATING
+    end
+
+    if query_params['cuisine'].present?
```

Comment:
After using the `params` object, we usually refer to params as symbols (`:cuisine`, not `'cuisine'`)